src/CLUBB_core/clubb_api_module.F90

```diff
@@ -2607,6 +2607,7 @@ subroutine setup_parameters_api_single_col( &
              deltaz, params, nzmax, &
              grid_type, momentum_heights, thermodynamic_heights, &
              l_prescribed_avg_deltaz, &
+             l_linear_diffusion, & 
```

Comment:
Since the clubb_api_module has been changed, the host models (E3SM, etc.) will also need to change.  UWM will need to keep this in mind.